esiclient/v1/node_baremetal.py

```diff
@@ -0,0 +1,73 @@
+#   Licensed under the Apache License, Version 2.0 (the "License"); you may
+#   not use this file except in compliance with the License. You may obtain
+#   a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#   WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#   License for the specific language governing permissions and limitations
+#   under the License.
+
+import logging
+
+import openstack
+from osc_lib.command import command
+from osc_lib import exceptions
+from osc_lib.i18n import _
+
+# Initialize and turn on debug logging
+openstack.enable_logging(debug=False)
```

Comment:
this almost definitely should not be here. I think you may have turned on logging for the entire demo server by the way, as I see logs if I just log in and run 'openstack project list' - could you turn that off?